deploy/helm/values.yaml

```diff
@@ -690,7 +690,7 @@ policiesBundle:
   # -- repository of the policies bundle
   repository: aquasecurity/trivy-checks
```

Comment:
I think it's worth changing the default value in the future after the release of trivy-checks in DockerHub in case the use of embedded checks is disabled.

Reply:
yes, sure.